utilities/cert_utility.mak

```diff
@@ -54,15 +54,17 @@ AR=ar
 #export LD_LIBRARY_PATH=/usr/local/lib
 LDFLAGS= -L $(LOCAL_LIB) -lprotobuf -lgtest -lgflags -lpthread -L/usr/local/opt/openssl@1.1/lib/ -lcrypto -lssl
 
-dobj=	$(O)/cert_utility.o $(O)/certifier.pb.o $(O)/support.o $(O)/certifier.o \
-$(O)/certifier_proofs.o $(O)/simulated_enclave.o $(O)/application_enclave.o
+dobj =	$(O)/cert_utility.o $(O)/certifier.pb.o $(O)/support.o \
+        $(O)/certifier.o $(O)/certifier_proofs.o $(O)/simulated_enclave.o \
+        $(O)/application_enclave.o
 
-key_dobj=$(O)/key_utility.o $(O)/certifier.pb.o $(O)/support.o $(O)/certifier.o \
-$(O)/certifier_proofs.o $(O)/simulated_enclave.o $(O)/application_enclave.o
-
-mobj=	$(O)/measurement_init.o $(O)/certifier.pb.o $(O)/support.o $(O)/certifier.o \
-$(O)/certifier_proofs.o $(O)/simulated_enclave.o $(O)/application_enclave.o
+key_dobj = $(O)/key_utility.o $(O)/certifier.pb.o $(O)/support.o \
+           $(O)/certifier.o $(O)/certifier_proofs.o $(O)/simulated_enclave.o \
+           $(O)/application_enclave.o
 
+mobj = $(O)/measurement_init.o $(O)/certifier.pb.o $(O)/support.o \
+       $(O)/certifier.o $(O)/certifier_proofs.o $(O)/simulated_enclave.o \
+       $(O)/application_enclave.o
```

Comment:
Same for above two blocks .... Minor (unrelated) edit to indent the list of objects for improve readability.